shared/utils/combine-ovals.py

```diff
@@ -293,8 +301,10 @@ def checks(product, oval_version, oval_dirs):
                             continue
                         if check_is_loaded(already_loaded, filename, oval_version):
                             continue
-                        body.append(xml_content)
-                        included_checks_count += 1
+                        if get_oval_version_from_oval(xml_content) is not None and \
+                           Decimal(oval_version) >= Decimal(get_oval_version_from_oval(xml_content)):
+                            body.append(xml_content)
```

Comment:
I think the `get_oval_version_from_oval(xml_content)` should be called only once and the returned value saved to a variable, because now it reads the file twice.

Reply:
very good point @jan-cerny. This is fixed.